sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -221,6 +230,45 @@ void _CrtLeaveDbgReport(int reportType)
         _InterlockedDecrement(&_CrtInAssert);
 }
 
+EXTERN_C
+int __cdecl _CrtSetReportMode(int reportType, int reportMode)
+{
+    if (reportType >= _CRT_ERRCNT || reportType < 0)
+        return 0;
+
+    int oldReportMode = _CrtModeOutputFormat[reportType];
+    _CrtModeOutputFormat[reportType] = reportMode;
```

Comment:
if `reportMode == _CRTDBG_REPORT_MODE` then you should not update `_CrtModeOutputFormat`, this is only used to query the current value.

Reply:
Okay, fixed.